global.json

```diff
@@ -1,6 +1,6 @@
 {
   "projects": [ "src" ],
     "sdk": {
-    "version": "1.0.0-preview2-003131"
+    "version": "1.0.0-preview2-003121"
```

Comment:
Everyone should probably have the later version now. Was thinking of bumping dependencies to 4.3.0 also soon.